api/client/trust.go

```diff
@@ -37,7 +37,10 @@ import (
 	"github.com/docker/notary/tuf/data"
 )
 
-var untrusted bool
+var (
+	releasesRole = data.CanonicalTargetsRole + "/releases"
```

Comment:
That's something we need to actually double check (and ensure we're consistent about throughout notary). This is a TUF role and as such will always use "/" as the path separator. I'm not sure if `path.Join` will generate the correct output for TUF on Windows (when we add support for DCT there)